server/models/user.js

```diff
@@ -0,0 +1,73 @@
+const {pgPool} = require('../config/database');
+
+class User{
+    constructor(username=null, email=null, hash=null){
+        this.username = username;
+        this.email = email;
+        this.hash = hash;
+    }
+
+    async createUser(){
```

Comment:
It's generally a good programming practice to consider the class name when writing methods. Currently, a caller would call this method using `User.createUser()`. Instead, if you renamed this method to `create`, you'd call this method like `User.create()` which flows better with your API design. The same concept applies for the rest of the methods on the `User` class.

```suggestion
    async create(){
```